build-tools/packages/build-cli/docs/generate.md

```diff
@@ -355,6 +356,28 @@ EXAMPLES
 
 _See code: [src/commands/generate/releaseNotes.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/generate/releaseNotes.ts)_
 
+## `flub generate source-entrypoints`
```

Comment:
`generate sourceEntrypoints` to be consistent with the naming style of the other commands.